core/src/Exceptions/CoreErrorMessages.cs

```diff
@@ -91,5 +91,7 @@ internal class CoreErrorMessages
 
         public const string GetUserNameFailed = "Failed to get user name from the operating system.";
 
+        public const string GetTeamIdReturnedNull =
+           "TeamId returned null from iOS keychain. See https://aka.ms/msal-net-enable-keychain-access for more details on enabling keychain access.";
```

Comment:
What about:
Why do you get this error?
the application cannot access iOS keychain for application publisher (TeamId is null), whereas it's needed to enable Single Sign On between applications of the same publisher

How to remediate?
his iOS app configuration issue. See https://aka.ms/msal-net-enable-keychain-access for more details on enabling keychain access.